rpc/http.go

```diff
@@ -17,240 +17,24 @@
 package rpc
 
 import (
-	"bufio"
 	"bytes"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
-	"net"
 	"net/http"
 	"net/url"
-	"strconv"
 	"strings"
-	"time"
+	"sync"
 
-	"github.com/ethereum/go-ethereum/logger"
-	"github.com/ethereum/go-ethereum/logger/glog"
-	"gopkg.in/fatih/set.v0"
+	"github.com/rs/cors"
 )
 
 const (
-	httpReadDeadLine = 60 * time.Second // wait max httpReadDeadeline for next request
+	maxHTTPContentLength = 1024 * 64
```

Comment:
Are we sure this isn't too small? 64KB isn't that big of a message. Given that the RPC will only be used internally/manually by users, it's not really an attack vector to have a higher limit as the user can only screw its own node.